support/spring/src/main/java/org/apache/shiro/spring/web/config/AbstractShiroWebConfiguration.java

```diff
@@ -116,16 +120,18 @@ protected Cookie rememberMeCookieTemplate() {
                 rememberMeCookieMaxAge,
                 rememberMeCookiePath,
                 rememberMeCookieDomain,
-                rememberMeCookieSecure);
+                rememberMeCookieSecure,
+                rememberMeSameSite);
     }
 
-    protected Cookie buildCookie(String name, int maxAge, String path, String domain, boolean secure) {
+    protected Cookie buildCookie(String name, int maxAge, String path, String domain, boolean secure, Cookie.SameSiteOptions sameSizeOption) {
```

Comment:
This is technically a breaking change (if we want to merge back to 1.x)

We could add something like this, to retain backward compatibility:
```java
protected Cookie buildCookie(String name, int maxAge, String path, String domain, boolean secure) {
    return buildCookie(name, maxAge, path, domain, secure, sessionIdCookieSameSite);
}
```

Reply:
I'll add it. I forgot we turned off the API checker for the master branch. Good catch!